Watchman.Engine/Alarms/Defaults.cs

```diff
@@ -773,5 +773,26 @@ public static class Defaults
                 Namespace = AwsNamespace.Sqs
             }
         };
+
+        public static IList<AlarmDefinition> Dax = new List<AlarmDefinition>
+        { 
+            new AlarmDefinition
+            {
+                Enabled = true,
+                Name = "CPUUtilizationHigh",
+                Metric = "CPUUtilization",
+                Period = TimeSpan.FromMinutes(1),
+                EvaluationPeriods = 5,
+                Threshold = new Threshold
+                {
+                    ThresholdType = ThresholdType.Absolute,
+                    Value = 60
+                },
+                DimensionNames = new[] { "ClusterId" },
+                ComparisonOperator = ComparisonOperator.GreaterThanOrEqualToThreshold,
+                Statistic = Statistic.Average,
```

Comment:
Should this be max? as if this alarm is applied across many instances average could mask a single bad instance?